### DISCUSSION TITLE
How to get the tint image behavior to fire when back navigating to a contentpage

### DISCUSSION THREAD

**Discussion by mancda** [CommunityToolkit/Maui#713] [General]
My xaml

        <Image Source="{Binding Image}"   >
                <Image.Behaviors >
                    <toolkit:IconTintColorBehavior  TintColor="{StaticResource IconTintColor}"  />
                </Image.Behaviors>
            </Image>

If my content page is loaded by the navigationservice pushasync the tint behaviour works but if I back navigate to the contentpage using  

 var navigationService = DependencyService.Get<INavigationService>();
        await navigationService.NavigateBackAsync();

The tint behavior does not work.

Thanks for your help.

**Comment by pictos** [comment]
@mancda, 
Could you attach a small repro?

**Comment by CliffAgius** [comment]
Closed and answered
